### PR TITLE
FEATURE: Non hidden files with -profile test

### PR DESCRIPTION
**Pull request by maxulysse**
But we keep cached data in test_cache and all other subsequent tests

### CODE REVIEW

File: .github/workflows/pytest-workflow.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: pytest-workflow
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
   pull_request:
-    branches: [dev, master]
+    branches: [dev]
```
Comment (asp8200) [inline .github/workflows/pytest-workflow.yml:5]: Why is `master` being removed?  Don't you want the pytests to run on merge to master?

Comment (maxulysse) [reply .github/workflows/pytest-workflow.yml:5]: -> .github/workflows/pytest-workflow_release.yml

Comment (maxulysse) [reply .github/workflows/pytest-workflow.yml:5]: there might be some smarter way to do so, but the idea is to test with docker on dev, and add singularity and conda tests on master when we prepare for release

Comment (asp8200) [reply .github/workflows/pytest-workflow.yml:5]: okidoks

File: .github/workflows/pytest-workflow_release.yml [hunk 2]
```diff
@@ -0,0 +1,171 @@
+name: pytest-workflow
```
Comment (asp8200) [inline .github/workflows/pytest-workflow_release.yml:1]: Doesn't the "system" (GitHub?) require the workflows to have unique names?  (Perhaps not when they apply to different branches 🤔 )

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#1087](https://github.com/nf-core/sarek/pull/1087) - Back to dev
 - [#1087](https://github.com/nf-core/sarek/pull/1087) - Minor modules update
+- [#1088](https://github.com/nf-core/sarek/pull/1088) - Replace profile `test` by `test_cache` and add a `test` profile with non hidden files
```
Comment (asp8200) [inline CHANGELOG.md:14]:
```suggestion
- [#1088](https://github.com/nf-core/sarek/pull/1088) - Replace profile `test` by `test_cache` and add a `test` profile without hidden files
```

File: .github/workflows/pytest-workflow_release.yml (hunk 2 above)
Comment (maxulysse) [inline .github/workflows/pytest-workflow_release.yml:1]:
```suggestion
name: pytest-workflow-release
```

**Review by FriederikeHanssen — COMMENTED** [review]
sad but necessary. Only thing: we should update the usage docs. There is section on how to do your own testing https://nf-co.re/sarek/3.2.1/usage#how-to-test-the-pipeline that actually is probably horribly out of date

File: .github/workflows/pytest-workflow_release.yml [hunk 4]
```diff
@@ -0,0 +1,171 @@
+name: pytest-workflow-release
+# This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on:
+  push:
+    branches: [master]
+  pull_request:
```
Comment (maxulysse) [inline .github/workflows/pytest-workflow_release.yml:6]:
```suggestion
```

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**